app/components/conference-session.js

```diff
@@ -5,24 +5,27 @@ import ENV from 'emberconf/config/environment';
 import moment from 'moment';
 
 export default class extends Component {
+  /*
+   * This does not seem like it should be needed. We expected that autotrack
+   * would work because the root `now` on the controller was tracked.
+   */
+  @tracked args;
```

Comment:
This seems weird :-/ /cc @pzuraq @rwjblue is this as expected?

Reply:
Ah, I think this is an issue we meant to address but hadn't quite yet, as we didn't have `@tracked` yet. We should be able to add it directly to the property: https://github.com/glimmerjs/glimmer.js/blob/master/packages/@glimmer/component/addon/-private/component.ts#L184

I think we'll likely need to update the base class Glimmer.js, it shouldn't need the getter/setter anymore, since it'll inherit the tracked property from the main component definition: https://github.com/glimmerjs/glimmer.js/blob/master/packages/@glimmer/component/src/component.ts#L13

Reply:
Actually, I remember why we were ok with shipping this now - it should work because we're using `Ember.set`, and in theory that should integrate with `@tracked`. We also didn't want to rely on `@tracked` directly until its in stable. I think I know what's causing the issue, I'll update accordingly